services/incentive/src/main/java/org/cryptimeleon/incentive/services/incentive/error/BasketAlreadyPaidException.java

```diff
@@ -0,0 +1,7 @@
+package org.cryptimeleon.incentive.services.incentive.error;
+
+public class BasketAlreadyPaidException extends IncentiveServiceException {
+    public BasketAlreadyPaidException() {
+        super("");
```

Comment:
doesnt this mean that no message will be displayed?

Reply:
This is service side, all errors from this class will automatically be translated into a 400 with a fixed error message